libhb/hb.c

```diff
@@ -700,6 +703,8 @@ hb_buffer_t * hb_read_preview(hb_handle_t * h, hb_title_t *title, int preview)
             data += stride;
         }
     }
+
+:done
```

Comment:
Could you please move the semicolon in this patch, not the next?

Also considering the scope of the next patch, feel free to squash both commits.